1000$

```diff
@@ -2,61 +2,86 @@
 #include "balance.cpp"
 #include "randomEngine.cpp"
 
+struct Choices{
+    int number1;
+    int number2;
+    int value1;
+    int value2;
+};
+
 class NumberGame{
-    int computerChoice;
     int randNumber;
-    int humanChoice;
     Balance balance = Balance(1000, 1000);
     RandomEngine randomService;
 public:
     void run(){
-        humanChoice = getPlayerInput();
-        computerChoice = calcComputerTurn(humanChoice);
-        randNumber = randomService.getRandomNumber();
-            int absComputer = std::abs(computerChoice - randNumber);
-            int absHuman = std::abs(humanChoice - randNumber);
+        while(balance.getHumanScoreBalance() >= 0 || balance.getPcScoreBalance() >= 0) {
+            std::cout << balance.getHumanScoreBalance();
+            Choices ch;
+            getPlayerInput(ch);
+            calcComputerTurn(ch);
+            randNumber = randomService.getRandomNumber();
+
+            int absComputer1 = std::abs(ch.value1 - randNumber);
```

Comment:
You do not understand logic.

Example:
Input in console:
2
5
My value is 2 + 5 = 7
Computer numbers:
5 
7
Computer value: 9 + 7

Random create number 17

My value: 7
Computer Value: 16
17 - 7 = i have 10 missmatches point
17 - 16 = computer has 1 missmatch poin

Computer WIN